tests/acceptance/features/apiAntivirus/antivirusMain.feature

```diff
@@ -158,3 +158,71 @@ Feature: Antivirus basic
       | dav-path-version |
       | old              |
       | new              |
+
+  Scenario: A file cannot be overwritten with a file containing a virus via public upload
+    Given the administrator has enabled DAV tech_preview
+    And as user "Alice"
```

Comment:
why is this step used?

Reply:
@SwikritiT Kiran is asking about `L:164` `And as user "Alice"`